src/main/java/io/pinecone/clients/Pinecone.java

```diff
@@ -87,6 +91,7 @@ public IndexModel createServerlessIndex(String indexName, String metric, int dim
         CreateIndexRequestSpec createServerlessIndexRequestSpec = new CreateIndexRequestSpec().serverless(serverlessSpec);
 
         IndexModel indexModel = null;
+
```

Comment:
Should have left this comment for previous PR, sorry but For line#72:
1. if we are checking for null of line#69, then line#72 is redundant check, we don't need it.

Similarly, if line#54 is checking for null then line#57's null check is not needed